### ISSUE TITLE
Can't reach the results on the paper

### ISSUE THREAD

**Issue by ccnankai** [yangxue0827/R2CNN_FPN_Tensorflow#52]
Hi,
@yangxue0827 
I used my own data set for training, the test results are not good. I have trained 60,000 steps, and The loss function stops and does not reduced( stop at 0.9 ).
Can you give some Suggestions for improvement  to get better results？Which parameters need to be changed？

Thank you.

![image](https://user-images.githubusercontent.com/33537489/43322708-030549dc-91e2-11e8-9c22-e56cece093bc.png)

@yangJirui

**Comment by shihan19911126** [comment]
我也上遇到这样的情况，loss降到0.8在有就不再下降,请问你的问题解决了吗？有什么好的建议？

**Comment by shihan19911126** [comment]
@ccnankai 我看作者生成的total_loss图也是0.8左右，我用的resnet50可以降到0.73左右，效果还是resnet101好，虽然resnet101 loss高一些

**Comment by ccnankai** [comment]
@yangxue0827  ok，我主要在学习代码，多谢多谢。
